packages/babel-cli/bin/babel/file.js

```diff
@@ -100,7 +100,7 @@ module.exports = function (commander, filenames, opts) {
     results = [];
 
     _.each(filenames, function (filename) {
-      if (!fs.existsSync(filename)) return;
+      if (!pathExists.sync(filename)) return;
```

Comment:
Already made the changes on master. Haven’t pushed yet.

On Thu, Jun 25, 2015 at 11:11 PM, Mark Simulacrum
<notifications@github.com> wrote:

>> @@ -100,7 +100,7 @@ module.exports = function (commander, filenames, opts) {
>>      results = [];
>>  
>>      _.each(filenames, function (filename) {
>> -      if (!fs.existsSync(filename)) return;
>> +      if (!pathExists.sync(filename)) return;
> Should I open a new PR with the fix? Not sure how GitHub will work with new changes on the branch.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/babel/babel/pull/1832/files#r33310792